### PR TITLE
Add docs and samples for front-channel logout

### PR DESCRIPTION
**Pull request by jasonnutter**
Add docs and samples for how to implement single-sign out via front-channel logout.

### CODE REVIEW

File: lib/msal-browser/docs/logout.md [hunk 1]
```diff
@@ -86,6 +86,44 @@ await msalInstance.logoutPopup({
...
+    },
+    system: {
+        allowRedirectInIframe: true
+    }
+})
+
+// Automatically on page load
+msal.logout({
+    onRedirectNavigate: () => {
+        // Return false if you would like to stop navigation after local logout
```
Comment (jasonnutter) [inline lib/msal-browser/docs/logout.md:119]:
```suggestion
        // Return false to stop navigation after local logout
```

**Review by tnorling — APPROVED** [review]

File: lib/msal-browser/docs/logout.md [hunk 2]
```diff
@@ -86,6 +86,52 @@ await msalInstance.logoutPopup({
 });
 ```
 
+## Front-channel logout
```
Comment (pkanher617) [inline lib/msal-browser/docs/logout.md:89]: Can we add a TOC to this file? There are quite a few sections and some things might get lost if we don't have a reference to them at the top of the file.

**Review by pkanher617 — APPROVED** [review]
small comment about the docs, otherwise looks good

File: lib/msal-browser/docs/logout.md (hunk 2 above)
Comment (jasonnutter) [reply lib/msal-browser/docs/logout.md:89]: TOC are now added by Github automatically to all md files: https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

**Pull request merged**